### ISSUE TITLE
Troubleshooting Remote Session Disconnection Issue During Windows Login.

### ISSUE THREAD

**Issue by amoljagdalepucsd** [Ylianst/MeshCentral#5294]
We have a Windows machine with different users, including an admin, but currently, no user is logged in. This means that when I look at the computer screen, I see the usual Windows login screen showing a list of all the users at the bottom left corner. I can choose any user from this list, enter their password, and then access the system.

Now, while I'm trying to take desktop remote session of this system, I initiate a remote session. The first step involves a WebSocket call that goes for 'userSessions,' and I receive a response like this:
{
    "SessionId": 4,
    "StationName": "Console",
    "State": "Connected"
}

With the help of this inforamtion I am connected to the system through a desktop remote session. I can see the Windows login screen with the list of users on my remote desktop window. However, the challenge arises when I attempt to log in. After selecting a user, entering their password, and pressing Enter, the remote session gets disconnected immediately. This happens consistently, even after multiple attempts.

Given this situation, how can I effectively troubleshoot and resolve the issue to ensure that I can successfully log in through the remote session without facing disconnection?"**

My strong intuition is it's because of the sessionId (Windows user session ID) somehow in this response; it's giving an incorrect session ID and MeshCentral doesn't seem to handle it well.

**Comment by amoljagdalepucsd** [comment]
Thank you @si458 for reply and Yes control is restored.

**Comment by silversword411** [comment]
It's switching session captures from system to logged in user.

**Comment by amoljagdalepucsd** [comment]
Thanks you @silversword411 for answer.
but Is it possible to avoid this disconnection by doing some modification at code level ?
can you please guide me.

**Comment by si458** [comment]
@amoljagdalepucsd i dont believe the is a way of fixing this as this is more of an meshagent issue than a meshcentral issue
you can look at the meshagent source code here - https://github.com/Ylianst/MeshAgent

**Comment by amoljagdalepucsd** [comment]
Thank you @si458 for sharing your thoughts , I will check in meshagent source code.
